deploy/mobileclient_admin_role.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: mobileclient-admin
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+      - configmaps
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - mobile.k8s.io
+    resources:
+      - mobileclients
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - push.aerogear.org
+    resources:
+      - androidvariants
+      - iosvariants
+      - pushapplications
+      - unifiedpushservers
```

Comment:
Does MDC interact with the `unifiedpushservers`? I would have thought it only interacted with consumption crs?

Reply:
Good question.